Code/Tools/ProjectManager/Source/CreateAGemScreen.cpp

```diff
@@ -0,0 +1,446 @@
+        QLineEdit* originURLLineEdit = originURL->lineEdit();
+        originURLLineEdit->setPlaceholderText(tr("The primary repo for your Gem. i.e. http://www.mydomain.com"));
+        secondScreen->addWidget(originURL);
+
+        FormLineEditWidget* userDefinedGemTags = new FormLineEditWidget(tr("User-defined Gem Tags"), "");
+        secondScreen->addWidget(userDefinedGemTags);
+
+        FormFolderBrowseEditWidget* gemLocation = new FormFolderBrowseEditWidget(tr("Gem Location"), "");
+        QLineEdit* gemLocationLineEdit = gemLocation->lineEdit();
+        const char* gemLocationPlaceholderText = R"(C:\O3DE\Project\Gems\YourGem)";
+        gemLocationLineEdit->setPlaceholderText(tr(gemLocationPlaceholderText));
+        secondScreen->addWidget(gemLocation);
+
+        FormFolderBrowseEditWidget* gemIconPath = new FormFolderBrowseEditWidget(tr("Gem Icon Path"), "");
+        QLineEdit* gemIconPathLineEdit = gemIconPath->lineEdit();
+        gemIconPathLineEdit->setPlaceholderText(tr("Select Gem icon path"));
+        secondScreen->addWidget(gemIconPath);
+
+        FormLineEditWidget* documentationURL = new FormLineEditWidget(tr("Documentation URL"), "");
+        QLineEdit* documentationURLLineEdit = documentationURL->lineEdit();
+        documentationURLLineEdit->setPlaceholderText(
+            tr("Link to any documentation of your Gem i.e. https://o3de.org/docs/user-guide/gems/..."));
+        secondScreen->addWidget(documentationURL);
+
+        return secondScreenScrollArea;
+    }
+
+    QScrollArea* CreateAGemScreen::CreateThirdScreen()
+    {
+        QScrollArea* thirdScreenScrollArea = new QScrollArea();
+        thirdScreenScrollArea->setWidgetResizable(true);
+        thirdScreenScrollArea->setMinimumWidth(660);
+        thirdScreenScrollArea->setObjectName("createAGemRightPane");
+        QFrame* thirdScreenFrame = new QFrame();
+
+        QWidget* thirdScreenScrollWidget = new QWidget();
+        thirdScreenScrollArea->setWidget(thirdScreenScrollWidget);
+
+        QVBoxLayout* thirdScreen = new QVBoxLayout();
+        thirdScreen->setAlignment(Qt::AlignTop);
+        thirdScreen->addWidget(thirdScreenFrame);
+        thirdScreenScrollWidget->setLayout(thirdScreen);
+
+        QLabel* thirdRightPaneHeader = new QLabel(this);
+        thirdRightPaneHeader->setObjectName("rightPaneHeader");
+        thirdRightPaneHeader->setText(tr("Enter your Details"));
+        thirdScreen->addWidget(thirdRightPaneHeader);
+
+        m_creatorName = new FormLineEditWidget(tr("Creator Name*"), "", this);
+        QLabel* creatorNameErrorLabel = m_creatorName->getErrorLabel();
+        creatorNameErrorLabel->setObjectName("createAGemFormErrorLabel");
+        QLineEdit* creatorNameLineEdit = m_creatorName->lineEdit();
+        creatorNameLineEdit->setPlaceholderText(tr("John Smith"));
+        thirdScreen->addWidget(m_creatorName);
+
+        m_repositoryURL = new FormLineEditWidget(tr("Repository URL*"), "", this);
```

Comment:
```suggestion
        m_repositoryURL = new FormLineEditWidget(tr("Repository URL"), "", this);
```

I disagree that this field should be required, not all users will upload their gem to a repository. It should be validated if provided.

Reply:
This was decided by UX, leaving it in for now.